README.md

```diff
@@ -33,9 +33,6 @@ The goal of this project is to provide an open source library for building edito
 multiple authors, and multiple viewports.
 
 ## Requirements
-### Development requirements:
-
-- **swig** to generate language bindings (http://www.swig.org/svn.html)
```

Comment:
Why does this go away?

Reply:
This goes away because it was in there twice, if you look lower in the README.md you will the see the other occurance